### PR TITLE
backport:  v0.20 pr10

### PR DESCRIPTION
**Pull request by Munkybooty**
## Issue being fixed or feature implemented

Bitcoin Backports

## What was done?

Misc v20 Backports, check commit log to see changes.

## How Has This Been Tested?

Compilation and tested unit tests locally

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added or updated relevant unit/integration/functional/e2e tests
- [ ] I have made corresponding changes to the documentation

**For repository code-owners and collaborators only**
- [ ] I have assigned this pull request to a milestone

### CODE REVIEW

File: src/rpc/blockchain.cpp [hunk 1]
```diff
@@ -2623,8 +2623,9 @@ static UniValue getblockfilter(const JSONRPCRequest& request)
                 {RPCResult::Type::STR_HEX, "header", "the hex-encoded filter header"},
             }},
         RPCExamples{
-            HelpExampleCli("getblockfilter", "\"00000000c937983704a73af28acdec37b049d214adbda81d7e2a3dd146f6ed09\" \"basic\"")
-        },
+            HelpExampleCli("getblockfilter", "\"00000000c937983704a73af28acdec37b049d214adbda81d7e2a3dd146f6ed09\" \"basic\"")+
+        HelpExampleRpc("getblockfilter", "\"00000000c937983704a73af28acdec37b049d214adbda81d7e2a3dd146f6ed09\", \"basic\"")
```
Comment (knst) [inline src/rpc/blockchain.cpp:2627]: nit: missing space and different empty space

**Review by knst — CHANGES_REQUESTED** [review]
please, check this patch https://github.com/knst/dash/commit/ee6f3b7a648359bceaab91a89ae6c04559ec9f87 . Otherwise dash can't be compiled.

**Review by knst — DISMISSED** [review]
utACK

File: src/init.cpp [hunk 2]
```diff
@@ -2379,27 +2375,27 @@ bool AppInitMain(const CoreContext& context, NodeContext& node, interfaces::Bloc
 
     // ********************************************************* Step 10b: schedule Dash-specific tasks
 
-    node.scheduler->scheduleEvery(std::bind(&CNetFulfilledRequestManager::DoMaintenance, std::ref(netfulfilledman)), 60 * 1000);
-    node.scheduler->scheduleEvery(std::bind(&CMasternodeSync::DoMaintenance, std::ref(*::masternodeSync)), 1 * 1000);
-    node.scheduler->scheduleEvery(std::bind(&CMasternodeUtils::DoMaintenance, std::ref(*node.connman)), 60 * 1000);
-    node.scheduler->scheduleEvery(std::bind(&CDeterministicMNManager::DoMaintenance, std::ref(*deterministicMNManager)), 10 * 1000);
+    node.scheduler->scheduleEvery(std::bind(&CNetFulfilledRequestManager::DoMaintenance, std::ref(netfulfilledman)), 1min);
```
Comment (PastaPastaPasta) [inline src/init.cpp:2378]: kinda inconsistant, `1min` here, but `60s` later on

**Review by PastaPastaPasta — DISMISSED** [review]
utACK for merging via merge commit, although there were a few slight inconsistencies with like 60s vs 1min

**Comment by knst** [comment]
LGTM except mentioned formatting in [src/rpc/blockchain.cpp](https://github.com/dashpay/dash/pull/5083/files#diff-decae4be02fb8a47ab4557fe74a9cb853bdfa3ec0fa1b515c0a1e5de91f4ad0b)

**Review by UdjinM6 — APPROVED** [review]
utACK

**Review by UdjinM6 — DISMISSED** [review]
re-utACK

**Comment by knst** [comment]
@Munkybooty can you check commits in your force push?
Seems as accidentally you replaced valid commit with backport 18289 to empty commit
https://github.com/dashpay/dash/commit/bae9a8c8da882456993566a27fecbf2e7963b320
->
https://github.com/dashpay/dash/pull/5083/commits/f56477cc27970fade058d8083d8b9df51ef28471

**Review by knst — DISMISSED** [review]
utACK

**Review by UdjinM6 — DISMISSED** [review]
utACK

**Review by PastaPastaPasta — APPROVED** [review]
utACK for merging via merge commit

**Review by knst — APPROVED** [review]
re-utACK

**Pull request merged**